SMARTAGRIFOOD/vineyardData/schema.json

```diff
@@ -0,0 +1,194 @@
+{

```

Comment:
Much to comment on here please see a valid example [like this](https://github.com/smart-data-models/dataModel.Agrifood/blob/master/AgriFarm/schema.json) and the [contribution manual](https://bit.ly/contribution_manual) and try to make similarly 
1.- id attribute of the model is missing
2.- The x-ngsi attributes. Why are they here (are you mapping the automatically generated .yaml). Please don't include it. See contribution manual and the example.
3.- There are many attributes with are included in the [GSMA-Commons](https://github.com/smart-data-models/data-models/blob/master/common-schema.json) (you do  not need to list manually , just include the $ref like the example
4.- The attributes descriptions are not including Property/Relationship/Geopropery (see contribution manual pages 7-10)
5.- address is a very common and generic attribute,(is included in the  [Location-Commons](https://github.com/smart-data-models/data-models/blob/master/common-schema.json) by the $ref (see example) recode with another name
6.- There is not keyvalues payload to be validated by this schema.

Reply:
Thanks Alberto for the guidance. Yes, as you may figured, we started with YAML schema, and then transformed it into JSON, and then generated example data based on JSON schema. Some strange things appeared in the process, aparently.

A question on point 6 - is this referring to sample data that is provided?

Reply:
yes once you have a payload in the broker just make a -d keyvalues export and then you got the example.

This is the one validated with the schema.json